tests/JsonSchema/TypeFactoryTest.php

```diff
@@ -28,7 +30,16 @@ class TypeFactoryTest extends TestCase
      */
     public function testGetType(array $schema, Type $type): void
     {
-        $typeFactory = new TypeFactory();
+        $typeFactory = new TypeFactory(new ExtractorResourceNameCollectionFactory(
```

Comment:
Can't you use `this->prophesize(ResourceNameCollectionFactory::class)` instead? It will make the test easier to read

Reply:
Done.